pkg/cacheutil/memcached_client.go

```diff
@@ -73,6 +73,10 @@ type RemoteCacheClient interface {
 	// underlying async operation will fail, the error will be tracked/logged.
 	SetAsync(ctx context.Context, key string, value []byte, ttl time.Duration) error
 
+	// Del deletes a key from memcached.
+	// This operation is blocking and will return an error in case of failure.
+	Del(ctx context.Context, key string) error
```

Comment:
[nit] I suggest to be explict. We can afford 3 more characters ;)

```suggestion
	Delete(ctx context.Context, key string) error
```